test/test.ts

```diff
@@ -279,8 +279,37 @@ test('isValid', t => {
 	t.false(ow.isValid(true as any, ow.any(ow.string, ow.number)));
 });
 
+test('validate', t => {
+	const valueCauseError = 1 as any as string;
+
+	t.notThrows(() => {
+		const {error, value} = ow.validate('foo', ow.string);
+		t.true(error === null);
+		t.true(value === 'foo');
+	});
+
+	t.notThrows(() => {
+		const {error, value} = ow.validate('foo', 'label', ow.string);
+		t.true(error === null);
+		t.true(value === 'foo');
+	});
+
+	t.notThrows(() => {
+		const {error, value} = ow.validate(valueCauseError, ow.string);
+		t.true(error instanceof Error);
```

Comment:
Not sure but wouldn't it make more sense for `error` to be of type `string` and just contain the message of the error ?